packages/cms/src/features/release-notes/release-note-article.tsx

```
@@ -8,7 +8,7 @@ import type { Document } from "@contentful/rich-text-types";
 import React from "react";
 
 import type { ComponentReleaseNoteFieldsFragment as ReleaseNoteFields } from "../../lib/__generated/sdk";
-import { CtfRichText } from "../contentful/ctf-rich-text";
+import { CtfRichText, type EmbeddedEntryType } from "../contentful/ctf-rich-text";
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)